std/algorithm/sorting.d

```diff
@@ -986,6 +987,18 @@ template multiSort(less...) //if (less.length > 1)
     assert(points[1] == A(4, 1));
 }
 
+@safe unittest // issue 16179 (cannot access frame of function)
+{
+    auto arr = [[1, 2], [2, 0], [1, 0], [1, 1]];
+    auto lut = [2, 3, 1];
+
+    arr.multiSort!(
+        (a, b) => a[0] < b[0],
+        (a, b) => lut[a[1]] < lut[b[1]]
```

Comment:
btw this makes the test unnecessary complex, just multiplying with an integer of the function scope (as suggested in the bug report) will also trigger the frame access error